docs/quick_start/quick_start.md

```diff
@@ -31,7 +31,7 @@
 Currently, Apache Kyuubi maintains all its releases on our official [website](https://kyuubi.apache.org/releases.html).
 You can get the most recent stable release of Apache Kyuubi here:
 
-<a class="github-button" href="https://kyuubi.apache.org/releases.html" data-color-scheme="no-preference: light; light: dark; dark: light;" data-icon="octicon-download" data-size="large" aria-label="Download Kyuubi">Download</a>
```

Comment:
we can remove https://github.com/apache/incubator-kyuubi/pull/2443/files#diff-3a40f1d689268de6891f7d00b2c939bf0f67c9701bc8cfcc29869a134f02b472L18 too

Reply:
sorry, this's my first PR for Kyuubi.  I don't understand what you mean. Is it because of duplicate commit the same code?

Reply:
I don't find the remove way in https://github.com/apache/incubator-kyuubi/pull/2443/files#diff-3a40f1d689268de6891f7d00b2c939bf0f67c9701bc8cfcc29869a134f02b472L18

Reply:
I mean remove the line 18

Reply:
oh, thank you. The github can not jump to line 18 using this link.  I misunderstood.